### PR TITLE
Bug 2117646: Changing  field on any of routes in the openshift-console namespace wont trigger sync loop

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @TheRealJon

### CODE REVIEW

**Comment by TheRealJon** [comment]
Do we need to update or add any tests for this?

**Review by TheRealJon — APPROVED** [review]
Just the one question about whether or not we need to update or add any tests along with this change.

/lgtm

**Comment by jhadvig** [comment]
@TheRealJon thanks for comment. Updated the existing test. PTAL

File: test/e2e/util.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func patchAndCheckService(t *testing.T, client *framework.ClientSet, isOperatorM
 
 func patchAndCheckRoute(t *testing.T, client *framework.ClientSet, isOperatorManaged bool) error {
 	t.Logf("patching TargetPort on the console Route")
```
Comment (TheRealJon) [inline test/e2e/util.go:74]: Does this log statement need to be updated as well?

Comment (jhadvig) [reply test/e2e/util.go:74]: Yes, you are right. Im blind 🙈

**Pull request merged**